modules/Microsoft.AppConfiguration/configurationStores/readme.md

```diff
@@ -228,7 +228,17 @@ To use Private Endpoint the following dependencies must be deployed:
         // Example showing only mandatory fields
         {
             "subnetResourceId": "/subscriptions/<<subscriptionId>>/resourceGroups/validation-rg/providers/Microsoft.Network/virtualNetworks/sxx-az-vnet-x-001/subnets/sxx-az-subnet-x-001",
-            "service": "<serviceName>" // e.g. vault, registry, blob
+            "service": "<serviceName>", // e.g. vault, registry, blob
+            "ipConfigurations":[
```

Comment:
Is the IP configuration mandatory? You added the snippet to the 'area' with the header `// Example showing only mandatory fields`.

To this end, I believe it should be moved up a notch here, and in all other examples